file_organizer_v2/src/file_organizer/cli/dedupe.py

```
@@ -0,0 +1,614 @@
+#!/usr/bin/env python3
+"""
+Deduplication CLI - Interactive duplicate file detection and removal.
+
+This module provides a user-friendly command-line interface for finding and
+removing duplicate files using hash-based detection.
+"""
+
+import argparse
+import sys
+from pathlib import Path
+from typing import Dict, List, Optional
```

Ruff (0.14.13)

`typing.Dict` is deprecated, use `dict` instead

(UP035)